specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-03-01/virtualMachineScaleSet.json

```diff
@@ -1368,6 +1368,60 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/reapply": {
```

Comment:
>reapply [](http://example.com/codeflow?start=141&length=7)

"x-ms-long-running-operation-options" must be added to indicate how it is tracked. 

[https://github.com/Azure/azure-openapi-validator/blob/main/docs/post-operation-async-response-validation.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/post-operation-async-response-validation.md)

Reply:
Done.